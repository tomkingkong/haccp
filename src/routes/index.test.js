import React from 'react';
import { shallow } from 'enzyme';

import Routes from '.';

describe('Routes', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Routes />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});