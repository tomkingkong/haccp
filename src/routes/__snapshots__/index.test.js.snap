// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should match snapshot 1`] = `
<div>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/signup"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/profile"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/dashboard"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/companyinfo"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/products"
  />
  <Route
    component={[Function]}
    path="/plans"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/ingredients"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/receiving"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/inventory"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/processing"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/packaging"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/plans/summary"
  />
</div>
`;
